packages/block-editor/src/components/iframe/index.js

```diff
@@ -134,13 +134,26 @@ function Iframe( {
 			setIframeDocument( node.contentDocument );
 		};
 		let iFrameDocument;
+		let iFrameLoadTimeout;
+
 		// Prevent the default browser action for files dropped outside of dropzones.
 		function preventFileDropDefault( event ) {
 			event.preventDefault();
 		}
-		function onLoad() {
+		function onLoad( event, retry = 0 ) {
 			const { contentDocument, ownerDocument } = node;
 			const { documentElement } = contentDocument;
+
+			// Sometimes Safari fires the load event before the document is ready.
+			if ( contentDocument.body === null && retry < 3 ) {
+				retry++;
+				iFrameLoadTimeout = setTimeout(
+					() => onLoad( event, retry ),
+					100
+				);
+				return;
+			}
```

Comment:
This doesn't make sense to me. You say that this only fails for zoom-out mode, but the fix suggests it's iframe related, so it must break iframes regardless of zoom-out mode?

Reply:
Sorry, poor wording on my part. It is broken on the iframe regardless of the zoomed out mode, but it only visibly manifests itself with zoomed out mode toggled on because at that point the required tranform css is not applied as the `block-editor-iframe__html` class is missing. I have updated the title and description to better reflect this.

To further illustrate this, below on the left are the classes that have been applied to the iframe elements on Chrome, compared to Safari on the right, so given these missing classes ( admin-color-fresh wp-embed-responsive are also missing off body) I imagine there are wider impacts of this issue, but I haven't noticed anything specifically outside the zoom mode:

<img width="1793" alt="Screenshot 2024-04-09 at 12 40 52 PM" src="https://github.com/WordPress/gutenberg/assets/3629020/296f3e24-eb21-40a4-a643-3aff31725e69">